graphene_sqlalchemy/types.py

```diff
@@ -21,70 +24,91 @@
 from .utils import get_query, is_mapped_class, is_mapped_instance
 
 
+class ORMField(OrderedType):
+    def __init__(
+        self,
+        type=None,
+        prop_name=None,
```

Comment:
The underlying name for this on the SQLAlchemy `ColumnProperty` is `key`; I think it might be nice to avoid coming up with new names where possible and calling this `key` instead of `prop_name`. What do you think?

Reply:
How about `property` or `attribute`? Marshmallow uses `attribute` for the same purpose https://marshmallow.readthedocs.io/en/3.0/api_reference.html#module-marshmallow.fields

Reply:
They seem both fine to me but I think prefer `property` because it seems a bit more consistent with `sqlalchemy`. Also I don't think we should base our judgement too much on what  `marshmallow` does since it is not directly related to `graphene` or `sqlalchemy`.

The more I think about it the more I like the idea of prefixing the param name with `orm_` because it is consistent with how `ORMField` relates to `graphen.Field`. It also completely avoids any risk of collision with `graphene` parameters down the road. 

How does `orm_property` sound to you? If it's too long we can shorten it to `orm_prop`.

Reply:
I think `attribute` is a pretty well-known name both in Python itself (`__getattr__`, `getattr`), the library ecosystem (Marshmallow, [attrs](https://www.attrs.org/en/stable/)) or in SQLAlchemy itself (`InstrumentedAttribute`, `Mapper.column_attrs`, it's all over [the class mapping API](https://docs.sqlalchemy.org/en/13/orm/mapping_api.html), in which case `attribute` might be more consistent with SQLAlchemy than `property`). We even use `attribute` in the docstring you wrote here.

Another option could be `field`, since we already have `exclude_fields` and `only_fields`.

Or `name`, like how SQLAlchemy `Column()` takes an explicit `name` of the column you're creating a mapping for, letting you name the field on your model differently than the underlying column in the DB.

I think the `orm_` prefix is unnecessarily redundant and unlikely to collide with parameters down the line.

Reply:
As discussed offline, I renamed to `model_attr`.